basis/html/templates/chloe/compiler/compiler.factor

```diff
@@ -74,17 +79,38 @@ SYMBOL: string-context?
 
 ERROR: tag-not-allowed-here ;
 
+CONSTANT: self-closing-tags {
```

Comment:
Ahh, you know what, I forgot that ``in?`` just does a membership check on sequences:

```factor
M: sequence in? member? ; inline
```

So it's probably okay :-)

Or switch it, up to you.